docs/concepts/extend.md

```
@@ -813,10 +809,46 @@ API reference
     // ---cut-before---
-    const annotations = getAnnotations(state);
-    const myData = annotations?.[myKey];
+     const annotations = getAnnotations(state);
+     const myData = annotations?.[myKey];
     ~~~~
 
+    The raw storage protocol for annotations is internal.  Custom parsers
+    should read annotations through `getAnnotations()` instead of relying on
+    symbol-keyed state properties directly.
+
+### Functions from `@optique/core/extension`
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)